tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,10 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert len(items) == 0
+    assert items == [] # not sure if we need this here
```

Comment:
👍 In tests, I like the asserts to be as specific as we can make them. We are told this should return an empty list in this situation, and your check literally checks for an empty list. This ensures the type is correct (a list) and that it has no contents.

Checking that the length of `items` is 0 on line 36 only tell us that there was a collection with nothing in it, not that it was a list (we could be checking for the length of an empty dictionary on line 36 and the assert statement would pass, but the method should return a list). 
```suggestion
    assert items == []
```